Automattic/wp-calypso
# Refactor date-picker button tests to use @testing-library/react

#### Changes proposed in this Pull Request

This PR refactors the `date-button` component to use `@testing-library/react` instead of `enzyme`.

Part of #63409.

#### Testing instructions

Verify tests still pass: `yarn run test-client client/my-sites/backup/backup-date-picker/test/date-button.jsx`
